docusaurus/docs/dev-docs/preview.md

```diff
@@ -0,0 +1,305 @@
+        async handler(uid, { documentId, locale, status }) {
+          // Fetch the complete document from Strapi
+          const document = await strapi.documents(uid).findOne({ documentId });
+          
+          // Generate the preview pathname based on content type and document
+          const previewPathname = getPreviewPathname(uid, { locale, document });
+
+          // Use Next.js draft mode passing it a secret key and the content-type status
+          const urlSearchParams = new URLSearchParams({
+            url: getPreviewPathname(uid, { locale, document }),
+            secret: previewSecret,
+            status,
+          });
+          return `${clientUrl}/api/preview?${urlSearchParams}`;
+        },
+      },
+    },
+  };
+};
+```
+
+### 4. Set up the front-end preview route
+
+Setting up the front-end preview route is highly dependent on the framework used for your front-end application.
+
+For instance, [Next.js draft mode](https://nextjs.org/docs/app/building-your-application/configuring/draft-mode) and
+[Nuxt preview mode](https://nuxt.com/docs/api/composables/use-preview-mode) provide additional documentation on how to implement the front-end part in their respective documentations.
+
+If using Next.js, a basic implementation could be like in the following example taken from the [Launchpad](https://github.com/strapi/LaunchPad/tree/feat/preview) Strapi demo application:
+
+```typescript title="/next/api/preview/route.ts"
+import { draftMode } from "next/headers";
+import { redirect } from "next/navigation";
+
+export async function GET(request: Request) {
+  // Parse query string parameters
+  const { searchParams } = new URL(request.url);
+  const secret = searchParams.get("secret");
+  const url = searchParams.get("url");
+  const status = searchParams.get("status");
+
+  // Check the secret and next parameters
+  // This secret should only be known to this route handler and the CMS
+  if (secret !== process.env.PREVIEW_SECRET) {
+    return new Response("Invalid token", { status: 401 });
+  }
+
+  // Enable Draft Mode by setting the cookie
+  if (status === "published") {
+    draftMode().disable();
+  } else {
+    draftMode().enable();
+  }
+
+  // Redirect to the path from the fetched post
+  // We don't redirect to searchParams.slug as that might lead to open redirect vulnerabilities
+  redirect(url || "/");
+}
+```
+
+### Next steps
+
+Once the preview system is set up, you need to adapt your data fetching logic to handle draft content appropriately. This involves:
+
+1. Create or adapt your data fetching utility to check if draft mode is enabled
+2. Update your API calls to include the draft status parameter when appropriate
+
+The following, taken from the [Launchpad](https://github.com/strapi/LaunchPad/tree/feat/preview) Strapi demo application, is an example of how to implement draft-aware data fetching in your Next.js front-end application:
```

Comment:
Note that these links are pointing to the feat/preview branch of the launchpad repo, which will 404 [when we merge our PR](https://github.com/strapi/LaunchPad/pull/19) on the main branch. Hopefully next week after the CMS release